data/workflows/en/01_bootstrap_project.md

```diff
@@ -2,33 +2,38 @@
 title: "Bootstrap a project"
 ---
 
-> **TL;DR**
-> 
-> if you need a minimal project to start hacking quickly, use `dune init`. If you need a complete development environment that follows best practices, use `spin`.
-As the recommended build system for OCaml, Dune offers a command `dune init` to bootstrap new projects.
+[Dune](https://dune.readthedocs.io/en/stable/overview.html) is recommended for bootstrapping projects using `dune init`.
 
-Once you have successfully installed opam, you can install Dune with `opam install dune`.
+> dune init --help
+>
+> **dune init {library,executable,test,project} NAME [PATH]** initialize a
+> new dune component of the specified kind, named NAME, with fields
+> determined by the supplied options.
 
-This will install Dune's binary in your current Opam switch, so `dune` should now be in your `PATH`. If it not, you probably need to run `eval $(opam env)` to configure your current terminal with Opam environment.
+Once you have [installed opam](https://opam.ocaml.org/doc/Install.html), you can install Dune with `opam install dune`. This installs the Dune binary into your `opam` switch, so `dune` should now be in your `PATH`. If it is not, you probably need to run `eval $(opam env)` to configure your current terminal with your opam environment.
```

Comment:
We should definitely link to installing opam, perhaps we should direct people to our documentation page i.e. https://v3.ocaml.org/tutorials/up-and-running-with-ocaml?

Reply:
that's a great idea. how should we deal with the inner-linking URLs? pathnames only? get pathnames from a `Module`? `protocol:host:pathname`? you may already have prior art :)

Reply:
Nothing that fancy yet :)) A pathname should suffice for the moment thanks